librz/core/cbin.c

```diff
@@ -2974,6 +3013,16 @@ static int bin_sections(RzCore *r, PJ *pj, int mode, ut64 laddr, int va, ut64 at
 				build_hash_string(pj, mode, hashtypes, data, datalen);
 				free(data);
 			}
+			char* type = section_type_to_string(r->bin, section->type);
+			if(type) {
+				pj_ks(pj, "type", type);
+			}
+			free(type);
+			char* flag = section_flag_to_string(r->bin, section->flags);
```

Comment:
if you get a list but then here you just get the string again, nothing changed. the purpose of getting a list was so that you could return the list here and print it as `flags: ["alloc", "write", ....]`.